ingest-app/src/cmr/ingest/services/ingest_service.clj

```diff
@@ -397,6 +397,22 @@
           (dissoc :revision-date :transaction-id)
           (update-in [:revision-id] inc)))))
 
+(defn delete-variable
+  "Deletes a tag with the given concept id"
+  [context variable-key]
+  (let [existing-concept (fetch-variable-concept context variable-key)]
+    (mdb/save-concept
+      context
+      (-> existing-concept
+          ;; Remove fields not allowed when creating a tombstone.
+          (dissoc :metadata :format :provider-id :native-id :transaction-id)
```

Comment:
The comment is a little off. We may want to change "not allowed" to "that are unnecessary". The native-id and provider-id, etc. will be set based on the value of the previous concept that is deleted in metadata-db, so we don't have to pass them over from the ingest-app.